crystalBox.py

```diff
@@ -292,7 +292,22 @@ def dLimits(self,dMin,dMax):
         self.dMin=dMin
         self.dMax=dMax
         self.loadCif()
+        def jasmineFunction(self):
+        print("This is Jasmine's Function")
+    
```

Comment:
these lines should be deleted: they're duplicates and indentation is off.

Reply:
Sounds good, the duplicate seems to be from making changes and committing them more than once, so that is good feedback for me to see what it's doing. Do I delete it or did you?